### ISSUE TITLE
Model Selection 

### ISSUE THREAD

**Issue by kwade4** [PoonLab/OpenRDP#12]
I trained a Random Forest Classifier on the data (202 breakpoints) using an 80-20 test-train split and performed 5-fold cross validation. 

So far, the best performing RF model has a cross validation accuracy of about 0.65. The accuracy for the test set is a little better than a guess (0.56), but the training accuracy is very high (approx 0.98). 

It seems like the model is overfitting, so I will look into adding more data and see if the training and testing sets are balanced.  I will also
